### ISSUE TITLE
Retain winds in AABW and NADW formation to preserve overturning

### ISSUE THREAD

**Issue by dhruvbhagtani** [dhruvbhagtani/sfc-perturbation-expts#12]
The formation of two water masses, the NADW and AABW, occur in the subpolar regions, and are crucial to the sustenance of global overturning circulation ([Talley 2013](https://tos.org/oceanography/article/closure-of-the-global-overturning-circulation-through-the-indian-pacific-an)). In this experiment, we remove (or weaken) winds only in the tropics and subtropics, but keep the surface forcing in the subpolar and polar regions similar to control run. We can therefore, expect that the NADW and AABW are formed and could potentially result in a similar overturning as the control run, which can lead to similar amounts of buoyancy fluxes entering the ocean.

Potential issues: We will still lack a lot of heat input at the equator due to a lack of wind-driven upwelling, so there might not be any heat to transport to high latitudes even if we can somehow maintain a similar overturning as the control run.

**Comment by navidcy** [comment]
Why this belongs to `Model Improvements` project? From what I know, we can run this experiment straight away if we all feel like it. There is no additional model development needed for such experiment to be ready to go, isn't it?

**Comment by dhruvbhagtani** [comment]
<img width="1597" alt="Screen Shot 2021-10-14 at 5 43 57 pm" src="https://user-images.githubusercontent.com/29700296/137315234-3d3bc35a-5acf-4083-8634-be5fe10a4a59.png">

**Comment by dhruvbhagtani** [comment]
Here, we have winds everywhere except in a northern hemisphere latitude range (10 - 50 N). The output file is : https://github.com/dhruvbhagtani/varying-surface-forcing/blob/main/Stress_except_NANP/Diagnostics.ipynb

**Comment by dhruvbhagtani** [comment]
There is an even stronger heat loss at polar regions than the control run (leftmost figure). Associated with this is a stronger sub polar gyre too. 

So, by accident, we have stumbled upon a case where we have more heat loss, and an associated stronger sub polar gyre. However, note that our wind stress curl could also be playing a role, as our mask increases from 0 to 1 within 45 - 55 degree N. Maybe we could create another experiment, this time with the mask applied only till 40 degree N, and check whether we get a similar sub polar gyre?

<img width="1755" alt="Screen Shot 2021-10-14 at 5 49 41 pm" src="https://user-images.githubusercontent.com/29700296/137316061-cf15b6fd-d5f9-4163-8f13-964d69ff79a1.png">
<img width="1743" alt="Screen Shot 2021-10-14 at 5 47 38 pm" src="https://user-images.githubusercontent.com/29700296/137315781-bf163bbd-e4ec-419a-ad28-5f184ee36df8.png">

**Comment by dhruvbhagtani** [comment]
The AMOC and AABW match very closely with the control run, which is a very encouraging result. If we look at the MOC, we can clearly see a NADW source in the North Atlantic.

<img width="1516" alt="Screen Shot 2021-10-14 at 5 53 23 pm" src="https://user-images.githubusercontent.com/29700296/137316611-c6290ae3-4cf2-4a65-b417-5888ab4c1fdb.png">


<img width="1461" alt="Screen Shot 2021-10-14 at 5 53 26 pm" src="https://user-images.githubusercontent.com/29700296/137316586-1f1cb84e-0ece-40f0-a689-aac3dfa44c84.png">

**Comment by AndyHoggANU** [comment]
Yes, I agree - this is an interesting result and worth digging into more!

**Comment by AndyHoggANU** [comment]
Actually, would be good to see the full overturning streamfunction for the last decade of these 3 expts ...

( Actually, scrap this comment. I can see what I want in #13.)
